app/src/main/java/com/mmm/parq/fragments/DriverHomeFragment.java

```diff
@@ -42,9 +65,82 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                 .findFragmentById(R.id.map);
         mapFragment.getMapAsync(this);
 
+        mFindParkingButton = (Button) view.findViewById(R.id.findparkingbutton);
+        mFindParkingButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                queue = HttpClient.getInstance(getActivity().getApplicationContext()).getRequestQueue();
+                String url = "http://162.243.244.79/reservations";
```

Comment:
should probably put in some constants url file or something, so if/when we hook up the domain name its easy to change all the reqs